src/traces/parcats/colorbar.js

```diff
@@ -0,0 +1,52 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var isNumeric = require('fast-isnumeric');
+
+var Lib = require('../../lib');
+var Plots = require('../../plots/plots');
+var Colorscale = require('../../components/colorscale');
+var drawColorbar = require('../../components/colorbar/draw');
+
+
+module.exports = function colorbar(gd, cd) {
```

Comment:
This was simplified recently. Take a look at https://github.com/plotly/plotly.js/pull/2681/commits/fa8614750586a2901ccf65a520521bb570eb7026

Reply:
Done in https://github.com/jonmmease/plotly.js/pull/1/commits/71e212bd0b341aaa05aa59cdf6b7492e8472bb43

I also added a mock that shows the colorbar. But the colorbar doesn't seem to follow the `domain` of the trace.  Is this consistent with other trace types?

![screen shot 2018-08-08 at 3 38 00 pm](https://user-images.githubusercontent.com/15064365/43861122-e2620d96-9b23-11e8-9027-8f10337cbcc3.png)